bitcoin/electrs/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM rust:alpine3.13
+
+RUN apk add clang-dev build-base linux-headers
+RUN rustup component add rustfmt
+COPY ./electrs /workdir/electrs
+WORKDIR /workdir/electrs
+EXPOSE 3002
+ENV RUSTFLAGS -Ctarget-feature=-crt-static
+RUN cargo build --release --locked --bin electrs
+CMD ./target/release/electrs --cookie 'user:password' \
+	--http-addr 0.0.0.0:3002 \
+	--db-dir /workdir/electrs/data \
+	--daemon-rpc-addr $DAEMON_RPC_ADDR \
+	--network $NETWORK \
+	--daemon-dir /workdir/bitcoin-data/ \
+	--cors '*'
```

Comment:
I expect this is going to leave a big ol' container running. Assuming the `electrs` binary stands more or less alone, I'd use a multi-stage build to pull it to a clean container that only has the executable, and run the command there. See the way we do the [keep client build](https://github.com/keep-network/keep-core/blob/master/Dockerfile#L69-L80) for an example.

No need to block this PR on that, but it gives us a good stepping stone to eventually doing a build and publish of this ourselves so everyone doesn't have to do a local build.

Reply:
Tried the multi-stage build:
```
ENV RUSTFLAGS -Ctarget-feature=-crt-static
RUN cargo build --release --locked --bin electrs

FROM alpine:3.13

WORKDIR /workdir

COPY --from=electrsbuild /workdir/electrs/target/release/electrs .

CMD ./electrs --cookie 'user:password' \
	--http-addr 0.0.0.0:3002 \
	--db-dir /workdir/electrs/data \
	--daemon-rpc-addr $DAEMON_RPC_ADDR \
	--network $NETWORK \
	--daemon-dir /workdir/bitcoin-data/ \
	--cors '*'
```
it builds fine, but running the container gives me:
```
electrs      | Error loading shared library libstdc++.so.6: No such file or directory (needed by ./electrs)
electrs      | Error loading shared library libgcc_s.so.1: No such file or directory (needed by ./electrs)
electrs      | Error relocating ./electrs: _ZNSt12out_of_rangeC1ERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE10_M_replaceEmmPKcm: symbol not found
electrs      | Error relocating ./electrs: __cxa_free_exception: symbol not found
electrs      | Error relocating ./electrs: __cxa_begin_catch: symbol not found
electrs      | Error relocating ./electrs: _ZNSt6chrono3_V212system_clock3nowEv: symbol not found
electrs      | Error relocating ./electrs: _ZSt7getlineIcSt11char_traitsIcESaIcEERSt13basic_istreamIT_T0_ES7_RNSt7__cxx1112basic_stringIS4_S5_T1_EES4_: symbol not found
electrs      | Error relocating ./electrs: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE5rfindEPKcmm: symbol not found
electrs      | Error relocating ./electrs: _ZSt28_Rb_tree_rebalance_for_erasePSt18_Rb_tree_node_baseRS_: symbol not found
electrs      | Error relocating ./electrs: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE4findEPKcmm: symbol not found
electrs      | Error relocating ./electrs: _ZSt24__throw_out_of_range_fmtPKcz: symbol not found
electrs      | Error relocating ./electrs: _ZdaPv: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1119basic_istringstreamIcSt11char_traitsIcESaIcEED1Ev: symbol not found
electrs      | Error relocating ./electrs: _Znwm: symbol not found
electrs      | Error relocating ./electrs: _ZNSt8__detail15_List_node_base9_M_unhookEv: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1118basic_stringstreamIcSt11char_traitsIcESaIcEED1Ev: symbol not found
electrs      | Error relocating ./electrs: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE7compareEPKc: symbol not found
electrs      | Error relocating ./electrs: _ZNSt18condition_variableD1Ev: symbol not found
electrs      | Error relocating ./electrs: _ZSt20__throw_length_errorPKc: symbol not found
electrs      | Error relocating ./electrs: _ZNSt8ios_baseC2Ev: symbol not found
electrs      | Error relocating ./electrs: __cxa_guard_release: symbol not found
electrs      | Error relocating ./electrs: _ZdlPv: symbol not found
electrs      | Error relocating ./electrs: _ZNSo9_M_insertIlEERSoT_: symbol not found
electrs      | Error relocating ./electrs: _ZNSt18condition_variable4waitERSt11unique_lockISt5mutexE: symbol not found
electrs      | Error relocating ./electrs: _ZNKSt8__detail20_Prime_rehash_policy11_M_next_bktEm: symbol not found
electrs      | Error relocating ./electrs: _ZSt18_Rb_tree_decrementPKSt18_Rb_tree_node_base: symbol not found
electrs      | Error relocating ./electrs: _ZSt16__throw_bad_castv: symbol not found
electrs      | Error relocating ./electrs: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE12find_last_ofEPKcmm: symbol not found
electrs      | Error relocating ./electrs: _ZNSt6localeC1Ev: symbol not found
electrs      | Error relocating ./electrs: _Znam: symbol not found
electrs      | Error relocating ./electrs: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE7compareEmmPKc: symbol not found
electrs      | Error relocating ./electrs: _ZNSt6chrono3_V212steady_clock3nowEv: symbol not found
electrs      | Error relocating ./electrs: _ZNSt18condition_variable10notify_allEv: symbol not found
electrs      | Error relocating ./electrs: __cxa_allocate_exception: symbol not found
electrs      | Error relocating ./electrs: _ZSt20__throw_out_of_rangePKc: symbol not found
electrs      | Error relocating ./electrs: _ZNSt6thread20hardware_concurrencyEv: symbol not found
electrs      | Error relocating ./electrs: _ZNKSt5ctypeIcE13_M_widen_initEv: symbol not found
electrs      | Error relocating ./electrs: _ZSt29_Rb_tree_insert_and_rebalancebPSt18_Rb_tree_node_baseS0_RS_: symbol not found
electrs      | Error relocating ./electrs: _ZNSt18condition_variable10notify_oneEv: symbol not found
electrs      | Error relocating ./electrs: __cxa_throw: symbol not found
electrs      | Error relocating ./electrs: __cxa_guard_abort: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE8_M_eraseEmm: symbol not found
electrs      | Error relocating ./electrs: _ZSt17__throw_bad_allocv: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE9_M_appendEPKcm: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE9_M_createERmm: symbol not found
electrs      | Error relocating ./electrs: _ZSt18_Rb_tree_incrementPKSt18_Rb_tree_node_base: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE12_M_constructEmc: symbol not found
electrs      | Error relocating ./electrs: _ZSt16__ostream_insertIcSt11char_traitsIcEERSt13basic_ostreamIT_T0_ES6_PKS3_l: symbol not found
electrs      | Error relocating ./electrs: __cxa_end_catch: symbol not found
electrs      | Error relocating ./electrs: __cxa_guard_acquire: symbol not found
electrs      | Error relocating ./electrs: _ZNKSt8__detail20_Prime_rehash_policy14_M_need_rehashEmmm: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1115basic_stringbufIcSt11char_traitsIcESaIcEE7_M_syncEPcmm: symbol not found
electrs      | Error relocating ./electrs: _ZNSt6localeD1Ev: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE4swapERS4_: symbol not found
electrs      | Error relocating ./electrs: _ZNSt8ios_base4InitC1Ev: symbol not found
electrs      | Error relocating ./electrs: _ZSt24__throw_invalid_argumentPKc: symbol not found
electrs      | Error relocating ./electrs: _ZNSt8ios_baseD2Ev: symbol not found
electrs      | Error relocating ./electrs: _ZNSt13runtime_errorC1EPKc: symbol not found
electrs      | Error relocating ./electrs: _ZNSt8__detail15_List_node_base7_M_hookEPS0_: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1119basic_ostringstreamIcSt11char_traitsIcESaIcEED1Ev: symbol not found
electrs      | Error relocating ./electrs: _ZSt19__throw_logic_errorPKc: symbol not found
electrs      | Error relocating ./electrs: _ZNSt6thread6_StateD2Ev: symbol not found
electrs      | Error relocating ./electrs: _ZSt18_Rb_tree_incrementPSt18_Rb_tree_node_base: symbol not found
electrs      | Error relocating ./electrs: _ZSt25__throw_bad_function_callv: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE9push_backEc: symbol not found
electrs      | Error relocating ./electrs: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE7compareEmmRKS4_: symbol not found
electrs      | Error relocating ./electrs: _ZNSt8__detail15_List_node_base11_M_transferEPS0_S1_: symbol not found
electrs      | Error relocating ./electrs: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE5rfindEcm: symbol not found
electrs      | Error relocating ./electrs: _ZSt18_Rb_tree_decrementPSt18_Rb_tree_node_base: symbol not found
electrs      | Error relocating ./electrs: _ZNSt9basic_iosIcSt11char_traitsIcEE4initEPSt15basic_streambufIcS1_E: symbol not found
electrs      | Error relocating ./electrs: _ZNSt6thread15_M_start_threadESt10unique_ptrINS_6_StateESt14default_deleteIS1_EEPFvvE: symbol not found
electrs      | Error relocating ./electrs: _ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE4findEcm: symbol not found
electrs      | Error relocating ./electrs: __cxa_throw_bad_array_new_length: symbol not found
electrs      | Error relocating ./electrs: _ZNSt16invalid_argumentC1ERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE: symbol not found
electrs      | Error relocating ./electrs: _ZNSt6thread4joinEv: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEaSEOS4_: symbol not found
electrs      | Error relocating ./electrs: _ZNSt18condition_variableC1Ev: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE7reserveEm: symbol not found
electrs      | Error relocating ./electrs: _ZNSo9_M_insertImEERSoT_: symbol not found
electrs      | Error relocating ./electrs: _ZNSt9basic_iosIcSt11char_traitsIcEE5clearESt12_Ios_Iostate: symbol not found
electrs      | Error relocating ./electrs: _ZNSolsEi: symbol not found
electrs      | Error relocating ./electrs: __cxa_rethrow: symbol not found
electrs      | Error relocating ./electrs: _Unwind_Resume: symbol not found
electrs      | Error relocating ./electrs: _ZNSt6thread6detachEv: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE9_M_mutateEmmPKcm: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE9_M_assignERKS4_: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE7replaceEmmPKcm: symbol not found
electrs      | Error relocating ./electrs: _ZSt20__throw_system_errori: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE6appendEPKc: symbol not found
electrumx    | Generating a RSA private key
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEC1EPKcRKS3_: symbol not found
electrs      | Error relocating ./electrs: _ZSt9terminatev: symbol not found
electrs      | Error relocating ./electrs: _ZNSo9_M_insertIdEERSoT_: symbol not found
electrs      | Error relocating ./electrs: _ZSt11_Hash_bytesPKvmm: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE6resizeEmc: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE14_M_replace_auxEmmmc: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv117__class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__si_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTINSt6thread6_StateE: symbol not found
electrs      | Error relocating ./electrs: _ZTINSt6thread6_StateE: symbol not found
electrs      | Error relocating ./electrs: _ZTINSt6thread6_StateE: symbol not found
electrs      | Error relocating ./electrs: _ZTINSt6thread6_StateE: symbol not found
electrs      | Error relocating ./electrs: _ZTINSt6thread6_StateE: symbol not found
electrs      | Error relocating ./electrs: _ZTINSt6thread6_StateE: symbol not found
electrs      | Error relocating ./electrs: _ZTINSt6thread6_StateE: symbol not found
electrs      | Error relocating ./electrs: _ZTINSt6thread6_StateE: symbol not found
electrs      | Error relocating ./electrs: _ZTINSt6thread6_StateE: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: __cxa_pure_virtual: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv121__vmi_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv121__vmi_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv121__vmi_class_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__function_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__function_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv120__function_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv119__pointer_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv119__pointer_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZTVN10__cxxabiv119__pointer_type_infoE: symbol not found
electrs      | Error relocating ./electrs: _ZNSt16invalid_argumentD1Ev: symbol not found
electrs      | Error relocating ./electrs: _ZTISt12out_of_range: symbol not found
electrs      | Error relocating ./electrs: _ZTVNSt7__cxx1118basic_stringstreamIcSt11char_traitsIcESaIcEEE: symbol not found
electrs      | Error relocating ./electrs: _ZTVNSt7__cxx1115basic_stringbufIcSt11char_traitsIcESaIcEEE: symbol not found
electrs      | Error relocating ./electrs: _ZTTNSt7__cxx1119basic_ostringstreamIcSt11char_traitsIcESaIcEEE: symbol not found
electrs      | Error relocating ./electrs: _ZTVSt9basic_iosIcSt11char_traitsIcEE: symbol not found
electrs      | Error relocating ./electrs: _ZTVNSt7__cxx1119basic_ostringstreamIcSt11char_traitsIcESaIcEEE: symbol not found
electrs      | Error relocating ./electrs: _Unwind_FindEnclosingFunction: symbol not found
electrs      | Error relocating ./electrs: _Unwind_GetIP: symbol not found
electrs      | Error relocating ./electrs: _ZNSt12out_of_rangeD1Ev: symbol not found
electrs      | Error relocating ./electrs: _Unwind_Backtrace: symbol not found
electrs      | Error relocating ./electrs: _ZTTNSt7__cxx1119basic_istringstreamIcSt11char_traitsIcESaIcEEE: symbol not found
electrs      | Error relocating ./electrs: _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEED1Ev: symbol not found
electrs      | Error relocating ./electrs: _ZNSt8ios_base4InitD1Ev: symbol not found
electrs      | Error relocating ./electrs: _ZTVSt15basic_streambufIcSt11char_traitsIcEE: symbol not found
electrs      | Error relocating ./electrs: _ZTVNSt7__cxx1119basic_istringstreamIcSt11char_traitsIcESaIcEEE: symbol not found
electrs      | Error relocating ./electrs: _ZTISt13runtime_error: symbol not found
electrs      | Error relocating ./electrs: _Unwind_GetCFA: symbol not found
electrs      | Error relocating ./electrs: _ZTTNSt7__cxx1118basic_stringstreamIcSt11char_traitsIcESaIcEEE: symbol not found
electrs      | Error relocating ./electrs: _ZNSt13runtime_errorD1Ev: symbol not found
electrs      | Error relocating ./electrs: _ZTISt16invalid_argument: symbol not found
electrs      | Error relocating ./electrs: __gxx_personality_v0: symbol not found
electrs      | Error relocating ./electrs: _ZTISt9exception: symbol not found
electrumx    | .................+++++
electrs exited with code 127
electrumx    | .................................................................................................................+++++
```
I think this has to do with a bunch of the libraries being dynamically linked in the build. You cool with leaving the larger image for now?

Reply:
Maybe adding just `gcc` will do the work?
`RUN apk add --no-cache gcc`

Reply:
Yep, looks like it worked for me locally.